pkg/services/control/service.proto

```diff
@@ -20,6 +20,9 @@ service ControlService {
     // Returns list that contains information about all shards of a node.
     rpc ListShards (ListShardsRequest) returns (ListShardsResponse);
 
+    // Returns list that contains information about all objects in a node.
+    rpc ListObjects (ListObjectsRequest) returns (ListObjectsResponse);
```

Comment:
have you considered streaming RPC? there can be millions or even billions objects, each of them at least is 64 bytes long